tests/integration/helpers/cq-height-test.ts

```diff
@@ -1,13 +1,13 @@
 import type { TestContext } from '@ember/test-helpers';
-import { render } from '@ember/test-helpers';
+import { getDeprecations, render } from '@ember/test-helpers';
 import { setupRenderingTest } from 'dummy/tests/helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import { module, test } from 'qunit';
 
 module('Integration | Helper | cq-height', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('can return a hash with default values', async function (this: TestContext, assert) {
+  test('provides a deprecation message', async function (this: TestContext, assert) {
```

Comment:
**reminder**

Delete this file in the future pull request, which will remove `{{cq-aspect-ratio}}`, `{{cq-height}}`, and `{{cq-width}}` helpers.